### ISSUE TITLE
Compilation error with BC19 artifacts on downloading symbols for System

### ISSUE THREAD

**Issue by pmatsconsulting** [microsoft/navcontainerhelper#1857]
**Describe the issue**
Compilation error with BC19 artifacts on downloading symbols for System
Command:  Compile-AppInBcContainer

**Scripts used to create container and cause the issue**
```
-Accept_Eula -Accept_Outdated -doNotCheckHealth -containerName TEMP-Build -auth Windows -updateHosts -includeCSide:False -memoryLimit 13G -doNotExportObjectsToText:True -enableSymbolLoading:False -includeTestToolkit:True -includeTestLibrariesOnly:True -doNotUseRuntimePackages:True -includeAL:False -assignPremiumPlan:True -isolation process -useBestContainerOS -artifactUrl https://bcinsider.azureedge.net/sandbox/19.0.23938.0/fr*** -EnableTaskScheduler:$False 
```

**Full output of scripts**
```
2021-04-02T06:41:06.6435556Z Creating container TEMP-Build...
2021-04-02T06:41:07.1188866Z BcContainerHelper is version 2.0.6
2021-04-02T06:41:07.1193170Z BcContainerHelper is running as administrator
2021-04-02T06:41:07.1211053Z Host is Microsoft Windows Server 2016 Standard - ltsc2016
2021-04-02T06:41:07.2256786Z Docker Client Version is 18.09.6
2021-04-02T06:41:07.2276799Z Docker Server Version is 18.09.6
2021-04-02T06:41:07.5187938Z Fetching all docker images
2021-04-02T06:41:07.6332283Z Fetching all docker volumes
2021-04-02T06:41:07.9825052Z Pulling image mcr.microsoft.com/businesscentral:10.0.14393.4046
2021-04-02T06:41:08.3850193Z 10.0.14393.4046: Pulling from businesscentral
2021-04-02T06:41:08.3897004Z Using image mcr.microsoft.com/businesscentral:10.0.14393.4046
2021-04-02T06:41:08.4927958Z Disabling Health Check (always report healthy)
2021-04-02T06:41:08.5108454Z Creating Container TEMP-Build
2021-04-02T06:41:08.5114032Z Version: 19.0.23938.0-FR
2021-04-02T06:41:08.5118688Z Style: sandbox
2021-04-02T06:41:08.5125197Z Multitenant: No
2021-04-02T06:41:08.5130245Z Platform: 18.0.23901.0
2021-04-02T06:41:08.5179940Z Generic Tag: 1.0.1.4
2021-04-02T06:41:08.5211344Z Container OS Version: 10.0.14393.4046 (ltsc2016)
2021-04-02T06:41:08.5217034Z Host OS Version: 10.0.14393.4046 (ltsc2016)
2021-04-02T06:41:08.5253031Z Using process isolation
2021-04-02T06:41:08.5270583Z Using locale fr-FR
2021-04-02T06:41:08.5284082Z Disabling the standard eventlog dump to container log every 2 seconds (use -dumpEventLog to enable)
2021-04-02T06:41:08.5304211Z Using license file E:\DevOps\Licenses\D365BC-V17-DEV-License.flf
2021-04-02T06:41:08.5600199Z Additional Parameters:
2021-04-02T06:41:08.5618665Z --volume "E:\DevOps\Agent\_work\63\s:C:\Source"
2021-04-02T06:41:08.5623741Z --env customNavSettings=EnableTaskScheduler=False
2021-04-02T06:41:08.5628564Z Files in C:\ProgramData\NavContainerHelper\Extensions\TEMP-Build\my:
2021-04-02T06:41:08.5657687Z - AdditionalOutput.ps1
2021-04-02T06:41:08.5663377Z - license.flf
2021-04-02T06:41:08.5668958Z - MainLoop.ps1
2021-04-02T06:41:08.5674802Z - SetupNavUsers.ps1
2021-04-02T06:41:08.5680379Z - SetupVariables.ps1
2021-04-02T06:41:08.5685878Z - updatehosts.ps1
2021-04-02T06:41:08.5691094Z Creating container TEMP-Build from image mcr.microsoft.com/businesscentral:10.0.14393.4046
2021-04-02T06:41:08.6954294Z 3b5d3106327e67756cb4e6802727b6f437cb662e4a65768b6c9fc40f43105512
2021-04-02T06:41:10.1895217Z Waiting for container TEMP-Build to be ready
2021-04-02T06:41:14.7012742Z Using artifactUrl https://bcinsider.azureedge.net/sandbox/19.0.23938.0/fr
2021-04-02T06:41:14.7012943Z Using installer from C:\Run\150-new
2021-04-02T06:41:14.7013401Z Installing Business Central
2021-04-02T06:41:14.7013571Z Installing from artifacts
2021-04-02T06:41:16.9744973Z Starting Local SQL Server
2021-04-02T06:41:18.1257721Z Starting Internet Information Server
2021-04-02T06:41:18.1257888Z Copying Service Tier Files
2021-04-02T06:41:18.1257929Z Copying PowerShell Scripts
2021-04-02T06:41:18.1257967Z Copying dependencies
2021-04-02T06:41:18.1258066Z Copying ReportBuilder
2021-04-02T06:41:19.2634020Z Importing PowerShell Modules
2021-04-02T06:41:19.2634214Z Determining Database Collation from c:\dl\sandbox\19.0.23938.0\fr\BusinessCentral-FR.bak
2021-04-02T06:41:53.7214645Z Changing Database Server Collation to French_100_CS_AS
2021-04-02T06:41:57.0814411Z Restoring CRONUS Demo Database
2021-04-02T06:41:58.2272156Z Setting CompatibilityLevel for CRONUS on localhost\SQLEXPRESS
2021-04-02T06:41:58.2272330Z Modifying Business Central Service Tier Config File for Docker
2021-04-02T06:41:58.2272371Z Creating Business Central Service Tier
2021-04-02T06:41:58.2273023Z Installing SIP crypto provider: 'C:\Windows\System32\NavSip.dll'
2021-04-02T06:41:58.2273439Z Copying Web Client Files
2021-04-02T06:41:58.2274344Z Copying Client Files
2021-04-02T06:41:58.2274416Z Copying ModernDev Files
2021-04-02T06:41:58.2274457Z Copying additional files
2021-04-02T06:41:58.2274494Z Copying ConfigurationPackages
2021-04-02T06:41:58.2274545Z Copying Test Assemblies
2021-04-02T06:41:58.2274583Z Copying Extensions
2021-04-02T06:41:58.2274620Z Copying Applications
2021-04-02T06:41:58.2274675Z Copying Applications.FR
2021-04-02T06:43:11.7974584Z Starting Business Central Service Tier
2021-04-02T06:43:11.7975065Z Importing license file
2021-04-02T06:43:12.9250215Z Stopping Business Central Service Tier
2021-04-02T06:43:12.9250369Z Installation took 118 seconds
2021-04-02T06:43:12.9250451Z Installation complete
2021-04-02T06:43:12.9250547Z Initializing...
2021-04-02T06:43:12.9250591Z Setting host.containerhelper.internal to 172.31.96.1 in container hosts file
2021-04-02T06:43:12.9250881Z Starting Container
2021-04-02T06:43:12.9250922Z Hostname is TEMP-Build
2021-04-02T06:43:14.0582860Z PublicDnsName is TEMP-Build
2021-04-02T06:43:14.0583379Z Using Windows Authentication
2021-04-02T06:43:14.0583686Z Modifying Service Tier Config File with Instance Specific Settings
2021-04-02T06:43:14.0583737Z Modifying Service Tier Config File with settings from environment variable
2021-04-02T06:43:14.0583840Z Setting EnableTaskScheduler to False
2021-04-02T06:43:37.0032341Z Starting Service Tier
2021-04-02T06:43:37.0032792Z Registering event sources
2021-04-02T06:43:39.5004882Z Creating DotNetCore Web Server Instance
2021-04-02T06:43:39.5005210Z Enabling Financials User Experience
2021-04-02T06:43:39.5005323Z Using license file 'c:\run\my\license.flf'
2021-04-02T06:43:39.5005370Z Import License
2021-04-02T06:43:40.4003103Z Creating http download site
2021-04-02T06:43:40.4003441Z Creating Windows user DevOpsAgent
2021-04-02T06:43:40.4003543Z Setting SA Password and enabling SA
2021-04-02T06:43:40.4003714Z WARNING: This license is not compatible with this version of Business Central.
2021-04-02T06:43:40.4003776Z Creating SUPER user
2021-04-02T06:43:46.1029142Z WARNING: This license is not compatible with this version of Business Central.
2021-04-02T06:43:46.1029483Z WARNING: This license is not compatible with this version of Business Central.
2021-04-02T06:43:46.1029674Z WARNING: This license is not compatible with this version of Business Central.
2021-04-02T06:43:47.2370545Z Assign Premium plan for TEMP-BUILD\DEVOPSAGENT
2021-04-02T06:43:47.2370788Z Container IP Address: 172.31.103.102
2021-04-02T06:43:47.2370829Z Container Hostname  : TEMP-Build
2021-04-02T06:43:47.2370870Z Container Dns Name  : TEMP-Build
2021-04-02T06:43:47.2371450Z Web Client          : http://TEMP-Build/BC/
2021-04-02T06:43:47.2371624Z Dev. Server         : http://TEMP-Build
2021-04-02T06:43:47.2372150Z Dev. ServerInstance : BC
2021-04-02T06:43:47.2372266Z Setting TEMP-Build to 172.31.103.102 in host hosts file
2021-04-02T06:43:47.2373945Z 
2021-04-02T06:43:47.2374085Z Files:
2021-04-02T06:43:47.2374129Z http://TEMP-Build:8080/ALLanguage.vsix
2021-04-02T06:43:47.2374208Z 
2021-04-02T06:43:47.2374252Z WARNING: You are running a container which is 131 days old.
2021-04-02T06:43:47.2374296Z Microsoft recommends that you always run the latest version of our containers.
2021-04-02T06:43:47.2374348Z 
2021-04-02T06:43:47.2374389Z Container Total Physical Memory is 16.0Gb
2021-04-02T06:43:47.2374426Z Container Free Physical Memory is 10.8Gb
2021-04-02T06:43:47.2374450Z 
2021-04-02T06:43:47.2374512Z Initialization took 35 seconds
2021-04-02T06:43:47.2376583Z Ready for connections!
2021-04-02T06:43:47.2390937Z Reading CustomSettings.config from TEMP-Build
2021-04-02T06:43:49.3997311Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:49.6223360Z Synchronizing Test Runner on default
2021-04-02T06:43:50.0831311Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:50.8047025Z App successfully synchronized
2021-04-02T06:43:50.8191700Z Installing Test Runner on default
2021-04-02T06:43:50.8290825Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:52.5214256Z App successfully installed
2021-04-02T06:43:52.5349679Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:53.2063519Z Publishing C:\ProgramData\NavContainerHelper\Extensions\TEMP-Build\22e352cb-f9b5-4351-b2ce-0805ae6936b1\Microsoft_Any_19.0.23938.0.app
2021-04-02T06:43:53.2166661Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:56.5724929Z Synchronizing Any on tenant default
2021-04-02T06:43:56.6205041Z Installing Any on tenant default
2021-04-02T06:43:56.6258963Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:56.8836826Z App Microsoft_Any_19.0.23938.0.app successfully published
2021-04-02T06:43:56.8978475Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:57.4820608Z Publishing C:\ProgramData\NavContainerHelper\Extensions\TEMP-Build\3519dc72-e9bb-4062-bce3-87e9e2eca0a0\Microsoft_Library Assert_19.0.23938.0.app
2021-04-02T06:43:57.4882742Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:57.8468033Z Synchronizing Library Assert on tenant default
2021-04-02T06:43:57.8793293Z Installing Library Assert on tenant default
2021-04-02T06:43:57.8844545Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:57.9874256Z App Microsoft_Library Assert_19.0.23938.0.app successfully published
2021-04-02T06:43:58.0039717Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:58.5779531Z Publishing C:\ProgramData\NavContainerHelper\Extensions\TEMP-Build\9e768497-e56e-48c6-aab4-c220520d59c6\Microsoft_Library Variable Storage_19.0.23938.0.app
2021-04-02T06:43:58.5843921Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:58.7024332Z Synchronizing Library Variable Storage on tenant default
2021-04-02T06:43:58.7385553Z Installing Library Variable Storage on tenant default
2021-04-02T06:43:58.7438501Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:58.8423169Z App Microsoft_Library Variable Storage_19.0.23938.0.app successfully published
2021-04-02T06:43:58.8575365Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:43:59.4524881Z Publishing C:\ProgramData\NavContainerHelper\Extensions\TEMP-Build\67bff243-9da3-402d-8372-09a898ae6afe\Microsoft_System Application Test Library_19.0.23938.0.app
2021-04-02T06:43:59.4591384Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:44:01.1524629Z Synchronizing System Application Test Library on tenant default
2021-04-02T06:44:01.2760064Z Installing System Application Test Library on tenant default
2021-04-02T06:44:01.2815521Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:44:01.6850813Z App Microsoft_System Application Test Library_19.0.23938.0.app successfully published
2021-04-02T06:44:01.7025445Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:44:02.2816600Z Publishing C:\ProgramData\NavContainerHelper\Extensions\TEMP-Build\92c8d382-0c6f-44b2-abe7-a0027e44a185\Microsoft_Tests-TestLibraries_19.0.23938.0.app
2021-04-02T06:44:02.2908854Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:44:19.2743236Z Synchronizing Tests-TestLibraries on tenant default
2021-04-02T06:44:21.3445062Z Installing Tests-TestLibraries on tenant default
2021-04-02T06:44:21.3503038Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:44:23.9224141Z App Microsoft_Tests-TestLibraries_19.0.23938.0.app successfully published
2021-04-02T06:44:23.9239825Z TestToolkit successfully imported
2021-04-02T06:44:23.9264712Z Container TEMP-Build successfully created...
```

**Output of command Compile-AppInBcContainer**
```
2021-04-02T06:46:22.9848374Z Compile App Calliope_xxxx.17.0.5305.0.app
2021-04-02T06:46:23.4933607Z Using Symbols Folder: E:\DevOps\Agent\_work\63\s\.alpackages
2021-04-02T06:46:38.4972698Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:46:38.5226723Z AVERTISSEMENT : This license is not compatible with this version of Business Central.
2021-04-02T06:46:39.0517975Z Downloading symbols: Microsoft_System_18.0.23901.0.app
2021-04-02T06:46:39.0543941Z Url : http://172.31.99.126:7049/BC/dev/packages?publisher=Microsoft&appName=System&versionText=17.0.0.0&tenant=default
2021-04-02T06:46:40.2110658Z Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
2021-04-02T06:46:40.2127363Z at <ScriptBlock>, <No file>: line 12
2021-04-02T06:46:40.2141610Z à Invoke-ScriptInBcContainer, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\2.0.6\ContainerHandling\Invoke-ScriptInNavContainer.ps1 : ligne 44
2021-04-02T06:46:40.2145310Z à Compile-AppInBcContainer, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\2.0.6\AppHandling\Compile-AppInNavContainer.ps1 : ligne 370
2021-04-02T06:46:40.2148911Z à <ScriptBlock>, E:\DevOps\Scripts\New\NAVDockerContainerManagement.ps1 : ligne 378
2021-04-02T06:46:40.2152567Z à <ScriptBlock>, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\2.0.6\AppHandling\Sort-AppFoldersByDependencies.ps1 : ligne 107
2021-04-02T06:46:40.2156151Z à Sort-AppFoldersByDependencies, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\2.0.6\AppHandling\Sort-AppFoldersByDependencies.ps1 : ligne 106
2021-04-02T06:46:40.2159735Z à Compile-NavContainerProjectApps, E:\DevOps\Scripts\New\NAVDockerContainerManagement.ps1 : ligne 329
2021-04-02T06:46:40.2163380Z à <ScriptBlock>, E:\DevOps\Scripts\New\NAVALRunBuild.ps1 : ligne 432
2021-04-02T06:46:40.2166934Z à <ScriptBlock>, E:\DevOps\Agent\_work\_temp\9a60d4dc-a68b-414c-8a31-fc8cadbfd505.ps1 : ligne 3
2021-04-02T06:46:40.2170432Z à <ScriptBlock>, <Aucun fichier> : ligne 1
2021-04-02T06:46:40.2600649Z Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
2021-04-02T06:46:40.2600912Z Au caractère C:\Program 
2021-04-02T06:46:40.2600966Z Files\WindowsPowerShell\Modules\BcContainerHelper\2.0.6\ContainerHandling\Invoke-ScriptInNavContainer.ps1:45 : 13
2021-04-02T06:46:40.2601069Z +             throw $_.Exception.Message
2021-04-02T06:46:40.2601113Z +             ~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-04-02T06:46:40.2601233Z     + CategoryInfo          : OperationStopped: (Unable to load ...re information.:String) [], RuntimeException
2021-04-02T06:46:40.2601492Z     + FullyQualifiedErrorId : Unable to load one or more of the requested types. Retrieve the LoaderExceptions propert 
2021-04-02T06:46:40.2601738Z    y for more information.
2021-04-02T06:46:40.2601970Z  
```
**Additional context**

**Comment by freddydk** [comment]
I have the same in my nextmajor pipelines this morning:
![image](https://user-images.githubusercontent.com/10775043/113391389-c173cf00-9393-11eb-9f63-921b34ae09bd.png)

Will investigate

**Comment by freddydk** [comment]
Shipped in 2.0.10
